Desenv

grsmth opened the pull request:
Feitos os testes na branch desenv, os bugs estão reportados nas issues.

FelipeSantanaC committed:
finalizacao do cadastro

FelipeSantanaC committed:
modal login

pablorenato1 committed:
Resolvendo o problema do Pop Up de Cadastro

pablorenato1 committed:
Update css para pop up cadastro

FelipeSantanaC committed:
css login

grsmth committed:
form_problem

grsmth committed:
form_ok

Falta dar o save no DB

pablorenato1 committed:
Adicionado Responsividade no Header e area de Pesquisa

grsmth committed:
Login and register working

FelipeSantanaC committed:
login route

grsmth committed:
model creation

-new model named local was created
-forms.py wasn't being used anymore, so it was deleted

grsmth committed:
load local data, fix model local

-local_data.csv and local_load_data_script.py are temp files that must be deleted further
-results.html still only a draft

grsmth committed:
api and search done

- bug on render the results.html
- search is not integrated on front end yet, but you can test it on localhost:8000/api/myapp/results/?search=KEY ( replacing KEY for your keyword)

pablorenato1 committed:
Update on Page Layouts

pablorenato1 committed:
Corrections on css

pablorenato1 reviewed (approved):
Todas as mudanças estão ok

grsmth committed:
Merge pull request #5 from FelipeSantanaC/searchengine

Searchengine

pablorenato1 committed:
Card creation in the home and results page

grsmth committed:
empty state implemented

bug on loading static files

pablorenato1 committed:
Fix footer positioning issue

grsmth committed:
path fixed

-updated local data

grsmth committed:
forms loading at every route

IMPORTANT: In authRst/settings.py add "myapp.context_processors.login_context" at the TEMPLATES list.
-registration, login, and logout are working
-search working
-local images aren't being displayed yet

pablorenato1 committed:
Update to display card images and add filtered score functionality to home page

pablorenato1 committed:
Tags in the results page added

pablorenato1 committed:
Merge pull request #6 from FelipeSantanaC/searchengine

Searchengine

DanCostaDev committed:
Correção da tela de cadastro e de login

FelipeSantanaC committed:
location filter

grsmth committed:
Merge remote-tracking branch 'origin/Cadastro' into desenv

grsmth committed:
Fixing conflicts.
modified:   authRst/myapp/static/css/style.css
modified:   authRst/myapp/templates/base.html

grsmth committed:
add missing line on base.html to load widget_tweaks

grsmth committed:
new models

-Login and register weren't working because the view URL was missing in base.html.
-Field is_active was included in the MyUser model.
-New models have been created to store users' information and preferences.

grsmth committed:
view created for additional form

-comp.html is a test-only file and it will be replaced in the final version
-db_input loads data with selectable preferences that will be displayed to the user

pablorenato1 committed:
Complementary pop up created

pablorenato1 committed:
Merge branch 'user_profile_model' of https://github.com/FelipeSantanaC/Turismo-Irrestrito into user_profile_model

grsmth committed:
list objects from db in html

-alter views.py comp.html to show local, dam, resources lists
-fix local images URL on local_data.csv
-alter the model.py local class attribute foto_url
-fix static logo path on popups

grsmth committed:
Merge pull request #8 from FelipeSantanaC/user_profile_model

improved pop-ups, creation of models and form for additional user information

grsmth committed:
form finished

-birthdate is in the right format now
-data is being saved on DB

pablorenato1 committed:
Merge pull request #9 from FelipeSantanaC/user_profile_model

form finished

DanCostaDev committed:
Alteração de Cores

Não foquei em ajeitar medidas e distâncias, vou fazer isso em uma task separada

pablorenato1 committed:
Merge pull request #10 from FelipeSantanaC/AlteracaoCores

Alteração de Cores

FelipeSantanaC committed:
profile screen

FelipeSantanaC committed:
screen about

pablorenato1 committed:
Updated filtering on the page results.

pablorenato1 committed:
Update navbar when the width is 800px or less

grsmth committed:
Merge pull request #11 from FelipeSantanaC/telaSobre

Criação da tela Sobre

DanCostaDev committed:
Merge branch 'desenv' into AlteracaoCores

DanCostaDev committed:
Merge pull request #12 from FelipeSantanaC/AlteracaoCores

Alteração na tela de resultados

pablorenato1 committed:
Updating the way that pop up is called

pablorenato1 committed:
Create a response in the front to wrong password/email

grsmth committed:
separate view login from register

-created  a new function to register only
-created a new file register.js
-modify HTML component ids
-added validation to email and password fields (that was supposed to be merged on my previous branch, i dont know why wasnt)

pablorenato1 committed:
Return a response to the front to show erro to the user

pablorenato1 committed:
Complete the task of handl errors from login and register new user

pablorenato1 committed:
Custom scroll bar added

grsmth committed:
Merge pull request #14 from FelipeSantanaC/handle-error-login

Handle error login

pablorenato1 committed:
Fix user registration redirection

FelipeSantanaC committed:
cadastro complementar

grsmth committed:
Merge pull request #15 from FelipeSantanaC/cadastroComplementar

Finalizando o cadastro complementar uma única vez para cada usuário

grsmth committed:
new app created

-.ipynp file was created with dataset generator and model training and testing
-In order to use ipynb file you must run:
pip install pandas
pip install -U scikit-learn

FelipeSantanaC committed:
model de preferencia

grsmth committed:
model persistence

- Created kmeans.joblib from the trained model

DanCostaDev committed:
Predict KMEANS

grsmth committed:
array normalization

thats all

FelipeSantanaC committed:
location recommendation

FelipeSantanaC committed:
Merge branch 'kmeans_model' of https://github.com/FelipeSantanaC/Turismo-Irrestrito into kmeans_model

grsmth committed:
Merge pull request #16 from FelipeSantanaC/kmeans_model

Finalizei a task de recomendação de local pelo cluster do usuário!

grsmth committed:
new app creation

-It was just created

grsmth committed:
Update models.py

- It was added two new classes: Rating and Post

grsmth committed:
api to get comments from a specific place

- Local id is used as parameter to filter

grsmth committed:
update db connection

grsmth committed:
update local score automatically on saving

- Alter model fields due an error in django migrations, it was appending a string "_id" at the end of field names user, local and rating.
- Tested.

pablorenato1 committed:
Merge pull request #19 from FelipeSantanaC/update-local-score-automatically

Update local score automatically
